### ISSUE TITLE
[linux Mint] 360 Xplorer shows up in devices but none of the buttons register

### ISSUE THREAD

**Issue by spoergsmaal** [YARC-Official/YARG#623]
hello, sorry for bothering you. i hope this is not some mistake on my own part.

issue: Clone Hero works perfectly fine with my 360 xplorer controller, YARG does not
for the record, https://gamepadtest.com/ successfully registers all inputs from the guitar controller distinctly

nightly (YARC launcher):
1. i start the nightly build through the appImage launcher
2. i click "Profiles" in the main menu
3. i click add profile
4. i click "connect" on the new profile
5. i click "RedOctane Guitar Hero X-plorer" on the "add device" list
6. i start a song
7. none of the buttons on the controller do anything

v0.11.1 (YARC launcher):
1. fatal error
2. "Failed to start YARG. Is it installed?\nOs { code: 13, kind: PermissionDenied, message: \"Permission denied\" }"

v0.11.1 (portable mode):
1. i run the executable
3. i click "add player"
4. i click "RedOctane Guitar Hero X-plorer"
5. green fret successfully binds to Y
6. red fret doesn't register at all, game is just waiting for an input or for me to cancel
7. yellow registers as "TriggerLeft2"
8. blue registers as "TriggerRight"
9. orange registers multiple inputs, lets me choose between "Select" and "Trigger"
10. strum up doesn't register at all
11. strum down doesn't register at all
12. whammy registers as "RotateX"
13. star power: i press select on the guitar and it registers as "ThumbRight"
14. tilt: i tilt the guitar, nothing registers
15. pause: registers as "Count"

Linux Mint 21.2 Cinnamon (5.8.4), kernel 5.15.0-91-generic
i7-3930K, nvidia GTX 1070

thank you for helping <3 sorry if it's my own fault

**Comment by EliteAsian123** [comment]
Hi, thanks for the report. Did you follow the Linux specific instructions in the README?

**Comment by spoergsmaal** [comment]
assuming that this is the readme you mean: https://github.com/YARC-Official/YARG#linux
had not gone through those steps before initial post, but now that I have, i'm still having issues :(

gamepadtest site still shows my controller as fully functional, but now the device doesn't even show up on the list in YARG (neither stable nor nightly)

**Comment by TheNathannator** [comment]
The Linux instructions unfortunately don't apply to this issue, need to write a new input backend altogether to fix this. The new Unity input system's controller support on Linux is not great lol

**Comment by RileyTheFox** [comment]
No. We still need to write a new backend for XInput on Linux. Unfortunately it's quite a large task to do and there have been higher priority things to work on

**Comment by AED-001** [comment]
Hi, I have the same issue as OP with my controller using santroller firmware. No strum up/strum down. Controller comes up fine in Steam's input tester and dpad up and dpad down register.

Just commenting to add a +1

**Comment by znoble360** [comment]
Adding my +1 in here too
